curriculum/js1/5.js

```diff
@@ -5,8 +5,10 @@
  * @param {function} b
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (X, F) => {
+  if(X===1) {
+    return F()}
+  return F()+solution(X-1,F);
```

Comment:
* Please use ensure that the first character of your variable names is lowercase (unless your variable is a constructor, which it's not here).
* Please add horizontal spaces around `=`, `===`, and after commas.
* This return statement does not make sense. Are you trying to add the function value to the value of the next call of the function?